### DISCUSSION TITLE
大家有什么好的开源项目，可以在这里面推荐哦

### DISCUSSION THREAD

**Discussion by biaochenxuying** [FrontEndGitHub/FrontEndGitHub#13] [Ideas]
## 推荐格式

- 项目标题
- 项目作者及简介
- 项目链接
- 推荐理由

经采纳的好项目，会放到这个仓库里面哦。

**Comment by Yohjisakamoto** [comment]
- **项目标题**：Tura — 本地开源 coding agent
- **项目作者及简介**：Tura-AI。Tura 是一个本地开源 coding agent，面向厌倦含糊技能宣传、缺乏证据的 token-saving 扩展，以及不先理解仓库就开始修改的 agent 的开发者。
- **项目链接**：[GitHub](https://github.com/Tura-AI/tura) · [官网](https://turaai.net/) · [公开 benchmark](https://turaai.net/benchmark)
- **推荐理由**：在公开的 20 个 DeepSWE v1.1 任务、60 次会话、GPT-5.6 SOL High 对比中，Tura Direct 比官方 Codex CLI High 少 83.5% 的总 token，验证成功率为 65.0% 对 60.0%；Tura Balanced 达到 80.0% 成功率，同时少用 49.6% token。完整方法、原始产物和适用范围请以 benchmark 页面为准。

披露：我是 Tura-AI 的关联方；本文由 AI 协助整理，并已人工核对。

**Comment by zhujinrui** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
